---
layout: page
title: Home
permalink: /
---

# Han Yuan, Ph.D.


üê¶ [@HY3952](https://twitter.com/HY3952) &nbsp;&nbsp;
üíº [LinkedIn](https://www.linkedin.com/in/han-yuan-0b909755/) &nbsp;&nbsp; 
üéì [Google Scholar](https://scholar.google.com/citations?user=FryLQFwAAAAJ)

{% include image.html url="images/profile.jpg" caption="" width="300px" height="250px" align="right" %}

## About Me

I am a machine learning scientist at Calico Life Sciences, where I study functional genomics using deep learning. My research focuses on developing DNA sequence‚Äìbased models to learn regulatory logic from functional genomic data, and on applying and interpreting these models to study gene regulatory networks, variant effects, and their roles in disease.

---

## Research Highlights

### Model Development

* **[Borzoi-PEFT](https://www.biorxiv.org/content/10.1101/2025.05.26.656171v2)** ‚Äì Parameter-efficient framework for transfer learning and regulatory network inference using pre-trained seq2fun models. (*Genome Biology*, 2026)
* **[Borzoi-PRIME](https://www.biorxiv.org/content/10.1101/2025.06.10.658961v1)** ‚Äì Foundational multitask seq2fun model that learns cell-type specific gene regulation in single cell atlases. (*bioRxiv*, 2025)
* **[Borzoi](https://www.nature.com/articles/s41588-024-02053-6)** ‚Äì Foundational multitask seq2fun model that learns cell-type specific gene regulation across thousands of tissue samples. (*Nature Genetics*, 2025)
* **[scBasset](https://www.nature.com/articles/s41592-022-01562-8)** ‚Äì Sequence-based convolutional neural network method to model scATAC data. (*Nature Methods*, 2022)
* **[BindVAE](https://link.springer.com/article/10.1186/s13059-022-02723-w)** ‚Äì Variational autoencoder approach to decode TF binding signals from open chromatin regions. (*Genome Biology*, 2022)
* **[BindSpace](https://www.nature.com/articles/s41592-019-0511-y)** ‚Äì Joint embedding approach to predict transcription factor binding. (*Nature Methods*, 2019)

### Biological Discoveries

* **Target Discovery** ‚Äì ML-model based target discovery in kidney and brain. (*Internal*)
* **[Worm aging atlas](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)00913-0)** ‚Äì A complete cell atlas of *C. elegans* aging. (*Cell Reports*, 2023)
* **[Genomic safe harbor](https://www.sciencedirect.com/science/article/pii/S0006497123003142)** ‚Äì CRISPR-based algorithm to identify genomic loci for safe and effective CAR-T therapy. (*Blood*, 2023)
* **[Multiomic view of Hayflick limit](https://elifesciences.org/articles/70283)** ‚Äì Identified key regulatory programs that drive replicative senescence. (*eLife*, 2022)

---

## Education

**2013 ‚Äì 2019** &nbsp;&nbsp; **Ph.D., Cornell University** in Computational Biology and Medicine.  
Thesis: *Modeling Large-scale Data Sets in Computational Genomics with Joint Learning*.  
Advisor: Dr. [Christina Leslie](https://www.mskcc.org/research/ski/labs/christina-leslie)

**2009 ‚Äì 2013** &nbsp;&nbsp; **B.S., Washington University in Saint Louis**, double major in Biology and Mathematics.  
GPA: 3.88/4.00.
