---
layout: page
title: Home
permalink: /
---

# Han Yuan, Ph.D.


üê¶ [@HY3952](https://twitter.com/HY3952) &nbsp;&nbsp;
üíº [LinkedIn](https://www.linkedin.com/in/han-yuan-0b909755/) &nbsp;&nbsp; 
üéì [Google Scholar](https://scholar.google.com/citations?user=FryLQFwAAAAJ)

{% include image.html url="images/profile.jpg" caption="" width="250px" height="250px" align="right" %}

I develop DNA sequence models to study gene regulation and disease. Currently at Calico Life Sciences, where my work combines deep learning and functional genomics to identify therapeutic targets and understand regulatory mechanisms.

Ph.D. in Computational Biology, Cornell University (2019).

---

## Selected Publications

<ul class="publication-list">
<li><strong><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656171v2">Borzoi-PEFT</a>‚Ä†</strong> ‚Äì Parameter-efficient framework for transfer learning and regulatory network inference using pre-trained seq2fun models. <em>(Genome Biology, 2026)</em></li>
<li><strong><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658961v1">Borzoi-PRIME</a></strong> ‚Äì Foundational multitask seq2fun model that learns cell-type specific gene regulation in single cell atlases. <em>(bioRxiv, 2025)</em></li>
<li><strong><a href="https://www.nature.com/articles/s41588-024-02053-6">Borzoi</a></strong> ‚Äì Foundational multitask seq2fun model that learns cell-type specific gene regulation across thousands of tissue samples. <em>(Nature Genetics, 2025)</em></li>
<li><strong><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(23)00913-0">Worm aging atlas</a>‚Ä†</strong> ‚Äì A complete cell atlas of <em>C. elegans</em> aging. <em>(Cell Reports, 2023)</em></li>
<li><strong><a href="https://www.sciencedirect.com/science/article/pii/S0006497123003142">Genomic safe harbor</a></strong> ‚Äì CRISPR-based algorithm to identify genomic loci for safe and effective CAR-T therapy. <em>(Blood, 2023)</em></li>
<li><strong><a href="https://www.nature.com/articles/s41592-022-01562-8">scBasset</a>‚Ä†</strong> ‚Äì Sequence-based convolutional neural network method to model scATAC data. <em>(Nature Methods, 2022)</em></li>
<li><strong><a href="https://link.springer.com/article/10.1186/s13059-022-02723-w">BindVAE</a></strong> ‚Äì Variational autoencoder approach to decode TF binding signals from open chromatin regions. <em>(Genome Biology, 2022)</em></li>
<li><strong><a href="https://elifesciences.org/articles/70283">Multiomic view of Hayflick limit</a></strong> ‚Äì Identified key regulatory programs that drive replicative senescence. <em>(eLife, 2022)</em></li>
<li><strong><a href="https://www.nature.com/articles/s41592-019-0511-y">BindSpace</a>‚Ä†</strong> ‚Äì Joint embedding approach to predict transcription factor binding. <em>(Nature Methods, 2019)</em></li>
</ul>

<div class="first-author-note">
‚Ä†First author and project lead
</div>

