---
layout: page
title: Research
permalink: /research/
---

### Characterize transcription factor binding using supervised embedding method
Transcription factor binding site prediction: Using in vitro binding profiles of transcription factors and genome-wide chromatin accessibility assays such as ATAC-seq or DNase-seq, we apply machine learning methods (such as topic model) to decode the transcription factor binding network in various cell types. 

### Predict cancer drug response using multitask learning
Using publicly available drug response and molecular characterization data sets of cancer cell lines, we developed a novel multi-task learning approach to predict drug sensitivity. While traditional approaches train a model for each drug independently, our model train one model for all drugs at once. Our model exploits the relationship between drugs to improve prediction performance. We implemented a trace norm regularized regression model that outperformed elastic net single-task learning model on all data sets.

### Collaboration projects
I engage in a number of collaborative projects to study cancer biology and immunology from a data-driven prospective. Meanwhile, together with my colleagues, we established a pipeline to process next-genereation sequencing data (RNA-seq, ChIP-seq, ATAC-seq) in an efficient and reproducible manner. These projects include: 
* identify target genomic regions for safe and efficient CRISPR editing in CAR-T cells, collaboration with Michel Sadelain lab;
* study the roles of BET proteins in response to viral infection, collaboration with Alexander Tarakhovsky lab;
* study characteristics of leukemia induced pluripotent stem cells, collaboration with Michael Kharas and Eirini Papapetrou lab;
* study expression changes in mouse regulatory T cells with gastrointestinal stromal tumors, collaboration with Ronald DeMatteo lab. 
